src/main/java/com/streamr/client/protocol/message_layer/GroupKeyRequest.java

```diff
@@ -18,7 +21,17 @@ public GroupKeyRequest(String requestId, String streamId, String rsaPublicKey, L
 
         this.requestId = requestId;
         this.publicKey = rsaPublicKey;
-        this.groupKeyIds = groupKeyIds;
+
+        this.groupKeyIds = new ArrayList<String>();
```

Comment:
Move this line to line 11 where variable is declared. Also remove redundant type declaration in `<String>` and use Java's diamond operator instead. Generic type is already defined on line 11.